fendermint/vm/interpreter/src/fvm/state/genesis.rs

```diff
@@ -97,11 +100,41 @@ where
         };
         let manifest = Manifest::load(&store, &manifest_data_cid, manifest_version)?;
 
+        // Load the custom actors bundle
+        let custom_actors_bundle_roots = load_car_unchecked(&store, custom_actor_bundle).await?;
+        let custom_actors_bundle_root = match custom_actors_bundle_roots.as_slice() {
+            [root] => root,
+            roots => {
+                return Err(anyhow!(
+                    "expected one root in actor bundle; got {}",
+                    roots.len()
+                ))
+            }
+        };
+
+        let (custom_actor_manifest_version, custom_actor_manifest_data_cid): (u32, Cid) =
+            match store.get_cbor(custom_actors_bundle_root)? {
+                Some(vd) => vd,
+                None => {
+                    return Err(anyhow!(
+                        "no manifest information in custom actor bundle root {}",
+                        bundle_root
+                    ))
+                }
+            };
```

Comment:
Can this and the load of the previous manifest root be moved into a utility function?